docs/user/getting-started.md

```diff
@@ -320,7 +322,7 @@ $ mkdir -p output && \
   sudo podman run --rm -it --privileged --pull=newer --security-opt label=type:unconfined_t \
     -v $(pwd)/output:/output -v /var/lib/containers/storage:/var/lib/containers/storage \
     quay.io/centos-bootc/bootc-image-builder:latest \
-    --type raw quay.io/${YOUR_QUAY_ORG}/centos-bootc-flightctl:v1
+    --type raw quay.io/${YOUR_QUAY_ORG}/centos-bootc-flightctl:v1 --local
```

Comment:
Please also align https://github.com/flightctl/flightctl/blob/main/docs/user/building-images.md?plain=1#L161-L167 with this.

Reply:
Done